navigation.yaml

```diff
@@ -1621,6 +1621,16 @@ ubuntu1604:
     - title: Azure
       path: /16-04/azure
 
+telco:
+  title: Telco
+  path: /telco
+
+  children:
+    - title: Overview
+      path: /telco
+    - title: Onboarding Network Functions
+      path: /telco/osm/onboarding-network-functions
```

Comment:
This doesn't seem right. There is no `/telco/osm` page. @pmahnke

Reply:
What do you think @pmahnke?

Reply:
I believe the `/telco/osm` page will come soon, @SirSamTumless